radiography/api/radiography.api

```diff
@@ -2,8 +2,6 @@ public final class com/squareup/radiography/BuildConfig {
 	public static final field BUILD_TYPE Ljava/lang/String;
 	public static final field DEBUG Z
 	public static final field LIBRARY_PACKAGE_NAME Ljava/lang/String;
-	public static final field VERSION_CODE I
-	public static final field VERSION_NAME Ljava/lang/String;
 	public fun <init> ()V
```

Comment:
Can we remove BuildConfig from the public api (not here, but in the lib)

Reply:
#36